lib/registries/addon/overview/styles.scss

```diff
@@ -78,3 +78,39 @@
     color: $primary-red;
     font-weight: bold;
 }
+
+.ResourcesWrapper {
+    display: inline-flex;
+    flex-direction: column;
+    justify-content: space-around;
+    background-color: #fff;
+    color: $primary-blue;
+    border-top: 1px solid #999;
+    padding-left: 8px;
+    width: 100%;
+}
+
+.ResourcesWrapper_title {
+    display: inline-flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+}
+
+.ResourcesWrapper h3 {
+    font-weight: 700;
+    color: #333;
+}
+
+.Registration_LeftNav {
+    background-color: #fff;
+    color: $primary-blue;
+}
```

Comment:
It looks like this class isn't used, so we can remove it

Reply:
Ok, this is now removed.